gridt/src/app/home/home.page.scss

```diff
@@ -15,6 +15,6 @@
 @media screen and (max-width: 576px) {
   .movement-card{
     width: 100%;
-    margin: 0;
+    margin: 5px 0;
```

Comment:
Have you considered using the `.ion-margin` class from [Ionic's CSS Utilities](https://ionicframework.com/docs/layout/css-utilities)?

Reply:
Currently there is a merge conflict due to recent changes. It should be easy to correct.

Reply:
Ionic's CSS utilities strangely don't change the margin. Only the hard-coded margin I've added works. 

Any suggestions?

Reply:
Well that is odd, because I seem to remember the padding does work that way. But this hard coded method is clearly the way to go then.

Reply:
Can we check a little more before we do the manual solution? I am in the hospital right now so I don't have access to a computer, but I feel like there should be a good reason why this is the case.

Reply:
Also, the Ionic margin classes are set to a default of 16px. You can't adjust them unless you go to the CSS variables and change it there. So if Ionic already requires us to apply other styling, maybe we should just do our homebrew solution;